src/register.js

```diff
@@ -0,0 +1,29 @@
+
+export function getProducts(products, code) {
+    for(let i = 0; i < products.length; i++) {
+        const product = products[i];
+        if(product.code === code) {
+            return product;
+        }
+    }
+    return null;
+} 
+
+export function lineTotes(thing, num) {
+    return Number((thing * num).toFixed(2));
+}
+
```

Comment:
re: naming - "thing" and "num" should be more explicit, ie "quantity" and "price". How would I know which is which with "num" and "thing"?